src/main/java/storagesystem/viewcontroller/inventory/InventoryController.java

```diff
@@ -0,0 +1,128 @@
+package storagesystem.viewcontroller.inventory;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.ChoiceBox;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.FlowPane;
+import storagesystem.model.*;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ResourceBundle;
+
+/**
+ * @author Pär Aronsson
+ */
+public class InventoryController implements Initializable {
+
+    private Organisation currentOrganisation;
+    private Team currentlySelectedTeam;
+    private User currentUser;
+    private List<IReservable> inventory;
+    private List<Team> currentUsersTeams = new ArrayList<>();
+    private ObservableList<String> teamNames = FXCollections.observableArrayList();
+
+
+    private int currentlySelectedTeamIndex;
+    private boolean isUserPartOfTeam;
+
+    private DetailedItemViewController detailView;
+
+    @FXML
+    FlowPane itemPane;
+
+    @FXML
+    AnchorPane rootPane;
+
+    @FXML
+    ChoiceBox teamChooser;
+
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        currentOrganisation = StoreIT.getCurrentOrganisation();
+        currentUser = StoreIT.getCurrentUser();
+        currentlySelectedTeam = StoreIT.getCurrentTeam();
+        isUserPartOfTeam = currentOrganisation.getUsersTeams(currentUser).size() > 0;
+        currentUsersTeams = StoreIT.getCurrentOrganisation().getUsersTeams(currentUser);
+        fillTeamAttributes();
+        refreshItems();
+    }
+
+    /**
+     * if a user is apart of a team. then this method fills the choicebox.
+     */
+    private void fillTeamAttributes() {
```

Comment:
Doesn't actually check if user is part of any team. If the user doesn't have any teams the program throws errors.